test/controller/lost-focus-controller.test.js

```diff
@@ -0,0 +1,190 @@
+import { expect } from '@bundled-es-modules/chai';
+import { fixture, defineCE, nextFrame } from '@open-wc/testing';
+import { html, LitElement } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import sinon from 'sinon/pkg/sinon-esm.js';
+import { LostFocusController } from '../../src/controllers/lost-focus-controller.js';
+import { dispatchCustomEvent } from '../../src/lib/events.js';
+
+describe('lost-focus-controller', () => {
+
+  const ce = defineCE(
+    class extends LitElement {
+      static get properties () {
+        return {
+          items: { type: Array },
+        };
+      }
+
+      constructor () {
+        super();
+        this.items = ['1', '2', '3'];
+
+        // eslint-disable-next-line no-new
+        new LostFocusController(this, '.item', (event) => {
+          dispatchCustomEvent(this, 'my-element:lostFocus', event);
+        });
+      }
+
+      removeItem (it) {
+        this.items = this.items.filter((item) => item !== it);
+      }
+
+      clear () {
+        this.items = [];
+      }
+
+      focusHeader () {
+        this.shadowRoot.querySelector('.header').focus();
+      }
+
+      focusItem (it) {
+        this.getItemElement(it)?.focus();
+      }
+
+      focusItemButton (it) {
+        this.getItemElement(it)?.querySelector('button').focus();
+      }
+
+      getItemElement (it) {
+        return this.shadowRoot.querySelector(`[data-item="${it}"]`);
+      }
+
+      render () {
+        return html`
+          <div class="header"></div>
+          ${repeat(this.items, (item) => item, (item) => {
+            return html`
+              <div tabindex="0" class="item" data-item="${item}"><button>${item}</button></div>
+            `;
+          })}
+        `;
+      }
+    },
+  );
+
+  /**
+   * @return {Promise<{element: Element, callback: Function}>}
+   */
+  const createElement = async () => {
+    const element = await fixture(`<${ce}></${ce}>`);
+
+    const callback = sinon.fake();
```

Comment:
We already have a simpler/smaller sinon alternative (hanbi), used in the smart manager tests.
Can you remove sinon and use hanbi please :wink:

The docs are simple but also because the API is simple.
https://github.com/43081j/hanbi